### ISSUE TITLE
Not Supported: Version 10.0.19041.1741

### ISSUE THREAD

**Issue by cowwoc** [stascorp/rdpwrap#1881]
Please provide an updated configuration for version 19041.1741

**Comment by sebaxakerhtc** [comment]
The offsets was added
https://github.com/sebaxakerhtc/rdpwrap.ini/commit/7175c06c1779c5aa6e34b14478ddcb96070cc3f2

**Comment by AlexNaBo** [comment]
Help please
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/8845239/termsrv.zip)
![Безымянный](https://user-images.githubusercontent.com/106975860/172184148-00276e51-ca2f-4382-8ca9-33524f3bb3da.png)

**Comment by sebaxakerhtc** [comment]
> Help please

cmd as admin:
net stop termservice
net start termservice

**Comment by AlexNaBo** [comment]
> 

Does not help

**Comment by AlexNaBo** [comment]
installed mod and it worked

**Comment by sashaqwert** [comment]
> [cowwoc](https://github.com/cowwoc) closed this as completed 4 days ago

@sebaxakerhtc please, reopen this, if this issue is first...

**Comment by WillyManson** [comment]
1

**Comment by lWX314616** [comment]
I tested it effectively
[rdpwrap.txt](https://github.com/stascorp/rdpwrap/files/8915647/rdpwrap.txt)
You should change 'txt' to 'ini' and replace original rdpwrap.ini.

**Comment by Leomemory** [comment]
> '

19044.1766 can use，thank you!

**Comment by sashaqwert** [comment]
@Leomemory 

> 19044.1766 can use

Yes. This is your Windows version. Version in RDPConf is same as the issue title.

**Comment by sebaxakerhtc** [comment]
If you want to try anything else with Single Session Per User - try to replace the offset to 117B9, but should work both

**Comment by nivcam** [comment]
![image](https://user-images.githubusercontent.com/28583638/177782351-a9fcb2f2-21f1-4372-b104-c908a4e4f7eb.png)
I have this error
meu sistema esta na 10.0.19043.1766
Would you help me?

**Comment by jikeidea** [comment]
I use the ini of https://github.com/stascorp/rdpwrap/issues/1881, but it doesn't work。
from #1928

**Comment by jikeidea** [comment]
![image](https://user-images.githubusercontent.com/43924085/178724460-c4a60523-126f-46ca-98bd-89cefc3e0e0d.png)
What's this mod?

**Comment by jikeidea** [comment]
can you help me? @binarymaster

**Comment by cruz-freitas** [comment]
alguem sabe me dizer quando eu dou o comando net stop termservice no cmd e ele fala que permissão negada erro 5 
como posso fazer

**Comment by sashaqwert** [comment]
@cruz-freitas 
> net stop termservice 

Is CMD launched as Administrator?

**Comment by sashaqwert** [comment]
@jikeidea 
> What's this mod?

https://github.com/sebaxakerhtc/rdpwrap/releases

**Comment by cmfish1990** [comment]
where is the config

**Comment by sashaqwert** [comment]
@cm
> where is the config

RDP_CnC.exe (in modded version)

**Comment by yungYee99** [comment]
> installed mod and it worked
"installed mod" what is mean

**Comment by sebaxakerhtc** [comment]
> > installed mod and it worked
> > "installed mod" what is mean

My modded version of RDP wrapper
https://github.com/sebaxakerhtc/rdpwrap/releases

**Comment by yungYee99** [comment]
> > > installed mod and it worked
> > > "installed mod" what is mean
> 
> My modded version of RDP wrapper https://github.com/sebaxakerhtc/rdpwrap/releases

get it thanks

**Comment by fernandokosta** [comment]
Please provide an updated configuration for version Windows 21H1 19043.1826. It's not listening 


![photo1658608723](https://user-images.githubusercontent.com/57380508/180623421-36979eec-aae4-41fc-8560-adb1169fb72e.jpeg)

**Comment by cruz-freitas** [comment]
@sashaqwert wow I had forgotten to put it as an administrator lol but it worked thank you!

**Comment by christian-westbrook** [comment]
I'm currently seeing not supported with this termsrv version

**Comment by sashaqwert** [comment]
> I'm currently seeing not supported with this termsrv version

Are you replaced the `rdpwrap.ini`?

**Comment by christian-westbrook** [comment]
> > I'm currently seeing not supported with this termsrv version
> 
> Are you replaced the `rdpwrap.ini`?

I have not. Should I?

**Comment by sashaqwert** [comment]
> I have not. Should I?

Yes, this is needed.

You can find offsets here (direct link to answer):  https://github.com/stascorp/rdpwrap/issues/1881#issuecomment-1145635621

Full INI available here: https://raw.githubusercontent.com/sebaxakerhtc/rdpwrap.ini/master/rdpwrap.ini

**Comment by christian-westbrook** [comment]
> > I have not. Should I?
> 
> Yes, this is needed.
> 
> You can find offsets here (direct link to answer): [#1881 (comment)](https://github.com/stascorp/rdpwrap/issues/1881#issuecomment-1145635621)
> 
> Full INI available here: https://raw.githubusercontent.com/sebaxakerhtc/rdpwrap.ini/master/rdpwrap.ini

Thank you so much! I'll try this out and report back.

**Comment by zzc1231** [comment]
![image](https://user-images.githubusercontent.com/22831061/184809869-5dad42dd-3c62-4acc-a7b7-a82fbf28812c.png)

**Comment by zzc1231** [comment]
> 

![image](https://user-images.githubusercontent.com/22831061/184809931-28ab411a-ab4d-484e-8010-8c6b3f8889eb.png)

**Comment by duduzai2019** [comment]
> 我有效地测试了它 [rdpwrap.txt](https://github.com/stascorp/rdpwrap/files/8915647/rdpwrap.txt) 你应该将'txt'改为'ini'并替换原来的rdpwrap.ini。

Thanks! I love you

**Comment by Dadz1** [comment]
> The offsets was added [sebaxakerhtc/rdpwrap.ini@7175c06](https://github.com/sebaxakerhtc/rdpwrap.ini/commit/7175c06c1779c5aa6e34b14478ddcb96070cc3f2)

Not working tried on 11/10/22
![Uploading 20221011_115931.jpg…]()

**Comment by SlightDust** [comment]
> 

it worked, thanks!
